apps/api_web/config/config.exs

```diff
@@ -39,7 +39,8 @@ config :api_web, :versions,
     "2019-02-12",
     "2019-04-05",
     "2019-07-01",
-    "2020-05-01"
+    "2020-05-01",
+    "2020-08-13"
```

Comment:
The date is somewhat arbitrary — going by past releases, we use the date at which the new version becomes the default. So I made it be A) a couple weeks in the future, and B) on a Thursday, since that's when our sprint ends. I'm not sure if we have a specific policy around choosing these dates.